lib/Controller/RoomController.php

```diff
@@ -305,6 +310,11 @@ protected function createOneToOneRoom($targetUserName) {
 			$room->addParticipant($targetUser->getUID(), Participant::OWNER);
 			$this->createNotification($currentUser, $targetUser, $room);
 
+			$this->backend->roomInvited($room, [
```

Comment:
I think I would move this into the methods of the `Room` class.
For better decoupling we could also trigger a hook? This would allow any app to listen to those events and provide a signaling backend?

Reply:
This is now updated to use events in the latest commit.